packages/now-cli/test/dev/integration.js

```diff
@@ -290,8 +328,16 @@ function testFixtureStdio(directory, fn) {
       });
 
       await readyResolver;
-      const helperTestPath = (...args) => testPath(t, port, ...args);
-      await fn(t, port, helperTestPath);
+
+      const helperTestPath = async (...args) => {
+        if (!skipDeploy) {
+          await testPath(t, `https://${deploymentUrl}`, ...args);
+        }
+        await testPath(t, `http://localhost:${port}`, ...args);
+      };
+      await fn(helperTestPath, t, port);
+    } catch (e) {
+      console.log(e);
```

Comment:
Is this one intentional? Won't the tests never fail since the error is not being thrown?

Reply:
Hmm, I can't remember exactly how I got here. I think I had a typo and `dev.unref` was undefined or something. I'll change it back 👍